corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -19,6 +19,7 @@
     </style>
 {% endblock %}
 {% block js %}{{ block.super }}
+    <script src="{% static 'moment/moment.js' %}"></script>
```

Comment:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/base.html#L187

i think we already include this library by default